### PR TITLE
[no merging]4225 Update tests for Vuetify feature: pagination

### PR DESCRIPTION
**Pull request by ngothikimthaoqb**

### CODE REVIEW

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Button.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.epam.jdi.light.vuetify.elements.common;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.vuetify.asserts.ButtonAssert;
+
+
+public class Button extends UIBaseElement<ButtonAssert>  {
```
Comment (omega790) [inline jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Button.java:8]: Please use VuetifyButton class

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ButtonAssert.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.epam.jdi.light.vuetify.asserts;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.vuetify.elements.common.Button;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class ButtonAssert extends UIAssert<ButtonAssert, Button> {
```
Comment (omega790) [inline jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ButtonAssert.java:10]: Please use VuetifyButtonAssert

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java [hunk 3]
```diff
@@ -44,6 +46,16 @@ public void circlePaginationTest() {
             circlePagination.has().value(page);
         }
         circlePagination.is().ended();
+        circlePagination.is().circle();
```
Comment (omega790) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java:49]: Please divide tests by features (for example, one for circle feature, one to check theme, one to check color, ....) and not duplicate checks

File: jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/PaginationAssert.java [hunk 4]
```diff
@@ -20,4 +25,91 @@ public PaginationAssert ended() {
...
+        jdiAssert(element().totalVisible(), Matchers.is(totalVisible));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' color is '{0}'")
+    public PaginationAssert color(String color, Boolean isMaterialColor) {
+        if (isMaterialColor) {
+            jdiAssert(element().activeButton().hasClass(color), Matchers.is(true));
+        } else {
+            jdiAssert(element().activeButton().has().borderColor(color), Matchers.is(color));
```
Comment (omega790) [inline jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/PaginationAssert.java:110]: Can be border color differ from background color?

Comment (ngothikimthaoqb) [reply jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/PaginationAssert.java:110]: color
When I add color Props for pagination, the border-color and background-color will change, so I think need to be add both CSS color for that

Comment (ngothikimthaoqb) [reply jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/PaginationAssert.java:110]: Hi @omega790, I fixed, please help me review on that

**Comment by MayaElf** [comment]
a new branch for the task was created and merged. task was completed.

**Pull request closed without merging**